doc/elements.md

```diff
@@ -1104,7 +1108,53 @@ Common reasons that trigger the display of this element are:
 
 #### See also
 
+- [`pl-question-panel` for displaying the question prompt.](#pl-question-panel-element)
 - [`pl-submission-panel` for changing how a submitted answer is displayed.](#pl-submission-panel-element)
+- [`pl-hide-in-panel` to hide contents in one or more display panels.](#pl-hide-in-panel-element)
+- [`pl-external-grader-results` for showing the results from an externally graded code question.](#pl-external-grader-results-element)
+
+
+-----
+
+
+## `pl-hide-in-panel` element
+
+Hide the contents in any of the panels ("question", "submission", or "answer").
+
+#### Sample Element
+
+```html
+<pl-hide-in-panel submission="True" answer="True">
+  This text will be hidden in the submission panel and answer panel.
+</pl-hide-in-panel>
+```
+
+#### Customizations
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`question` | boolean | False | Whether to hide the element contents in the question panel.
```

Comment:
One nit: I think booleans are listed as `true`/`false elsewhere in the doc?

Reply:
Good catch! In fact we were being totally inconsistent. I standardized everything on lowercase.